index.rst

```diff
@@ -1,40 +1,378 @@
-..
-  Content of technical report.
+:tocdepth: 1
 
-  See http://docs.lsst.codes/en/latest/development/docs/rst_styleguide.html
-  for a guide to reStructuredText writing.
+Technical notes (hereafter, *technotes*) are a new documentation medium for `LSST Data Management <http://dm.lsst.org>`_ staff to publish stand-alone documents that are native to the web.
+Technotes are written in reStructuredText, version controlled on GitHub, built with the same stack as DM's software documentation, and made universally citeable with `digital object identifiers (DOIs) <https://en.wikipedia.org/wiki/Digital_object_identifier>`_ provided through Zenodo_.
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+This document describes the Technote platform and will be updated as the platform evolves.
 
-  Use the following syntax for sections:
+DM team members can create a new technote today by following the instructions at https://github.com/lsst-sqre/lsst-technote-bootstrap
 
-  Sections
-  ========
+We currently use two technote series. The 'DMTN' is for general use within Data Management.
+The 'SQR' technote series is used for SQuaRE science quality verification and infrastructure work.
 
-  and
+.. _niche:
 
-  Subsections
-  -----------
+Technotes' Niche in the Data Management Communication Landscape
+===============================================================
 
-  and
+DM already has a myriad of communication channels.
+For conversations we use HipChat in real-time, and the `Community forum`_ for asynchronous long-form discussions, notices and community engagement.
+We track work on JIRA.
+We document our code in software docs (the platform for which is being re-imagined by SQuaRE).
+We describe the design and architecture of the DM subsystem in change-controlled design documents.
+We publish articles in journals and conference proceedings where appropriate.
+Finally, we have several Confluence wikis that traditionally served as a catch-all for DM documentation.
+Despite this array of platforms, a use case was still unserved and this technote platform was conceived to meet that need.
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+In SQuaRE, the Science Quality and Reliability Engineering group, we realized that we often wanted to write stand-alone documents to describe our work.
+These documents didn't fit the conversational nature of `Community forum`_ topics, nor did they qualify for DM's change-controlled design document series, or fit into academic journals or the arXiv_.
```

Comment:
How come "Community forum" has back tick quotes around it yet "arXiv" does not? They both seem to translate to external URLs using the same text style.